client/extensions/woocommerce/woocommerce-services/state/shipping-label/test/selectors.js

```diff
@@ -316,15 +316,21 @@ describe( 'Shipping label selectors', () => {
 			storeOptions: {
 				countriesData: {
 					US: {
-						CA: 'California',
+						name: 'US of A',
```

Comment:
Why do we need these changes to the tests?

Reply:
Because before, the structure of this test's state tree wasn't what was expected or what was returned by the real endpoint. The test passed because the `getCountriesData` function didn't operate on that data at all.